### PR TITLE
[Agent] Add team id and organize id to sender

### PR DESCRIPTION
**Pull request by yuanchaoa**
### This PR is for:

- Agent

#### Checklist
- [ ] Added unit test.
#### Backport to branches
- main
- 6.5

### CODE REVIEW

File: agent/src/sender/uniform_sender.rs [hunk 1]
```diff
@@ -120,7 +118,8 @@ impl<T: Sendable> Encoder<T> {
                 msg_type,
                 frame_size: 0,
                 version: 0,
-                sequence: 0,
+                team_id: 0,
+                organize_id: 0,
```
Comment (yuanchaoa) [inline agent/src/sender/uniform_sender.rs:122]: @lzf575  version是否需要修改

File: message/trident.proto [hunk 2]
```diff
@@ -247,6 +247,8 @@ message Config {
     optional uint32 vtap_id = 40;  // 限制在64000
     optional common.TridentType trident_type = 41 [default = TT_UNKNOWN];
     optional uint32 platform_sync_interval = 42 [default = 10];
+    optional uint32 team_id = 43 [default = 0];
+    optional uint32 organize_id = 44 [default = 0];
```
Comment (yuanchaoa) [inline message/trident.proto:251]: @sharang @jin-xiaofeng @SongZhen0704  这里加的两个字段

File: agent/src/sender/uniform_sender.rs [hunk 3]
```diff
@@ -83,21 +83,19 @@ struct Header {
     frame_size: u32, // tcp发送时，需要按此长度收齐数据后，再decode (FrameSize总长度，包含了 BaseHeader的长度)
     msg_type: SendMessageType,
 
-    version: u32,  // 用来校验encode和decode是否配套
-    sequence: u64, // 依次递增，接收方用来判断是否有丢包(UDP发送时)
-    vtap_id: u16,  // roze用来上报server活跃的VTAP信息
+    version: u32, // 用来校验encode和decode是否配套
+    team_id: u32,
+    organize_id: u32,
+    vtap_id: u16, // roze用来上报server活跃的VTAP信息
```
[the remark this answers is not in the document]
Comment (lzf575) [reply agent/src/sender/uniform_sender.rs:89]: ok

File: agent/src/sender/uniform_sender.rs (hunk 1 above)
Comment (lzf575) [reply agent/src/sender/uniform_sender.rs:122]: 一直都是0，不用改吧

File: message/trident.proto (hunk 2 above)
Comment (sharang) [reply message/trident.proto:251]: LGTM

**Review by jin-xiaofeng — APPROVED** [review]

**Pull request merged**